src/applications/edu-benefits/components/createOptOutApplicationStatus.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+const optOutForms = new Set(['22-0993']);
```

Comment:
I dunno if this is correct or necessary.

Reply:
Interesting question, as there are multiple types of edu forms, the edu wizard supports showing a user multiple saved forms. It doesn't seem like this form should be considered an edu form in the sense that it should be included in that list, nor that there will be multiple opt-out forms.

Reply:
Makes sense- removing them didn't impact any behavior or cause any errors.